core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -204,6 +235,8 @@ object AdminClient {
   val DefaultSendBufferBytes = 128 * 1024
   val DefaultReceiveBufferBytes = 32 * 1024
   val DefaultRetryBackoffMs = 100
+  val DefaultGroupQueryRetryIntervalMs: Int = 100
```

Comment:
Do we need this if we already have `DefaultRetryBackoffMs`? I can fix this when merging if you agree.